server/src/permissions/index.ts

```diff
@@ -13,8 +13,12 @@ export const authorizationError = {
   statusCode: 403,
 }
 
+// We want to skip authorization checks during integration tests
+const skipAuthorizations = process.env.ENV === 'test'
```

Comment:
I would have split this between constants and permissions:
```javascript
# constants.ts
const ENV = process.env.ENV

# permissions.ts
import {ENV} from "constants"
const skipAuthorization = ENV === "test"
const rules = skipAuthorization ? {} : { ...permissions resolvers...}
```

Reply:
I hesitated but you're right, it might actually be cleaner to do so